compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -179,6 +179,17 @@ impl<'local, 'interner> Interpreter<'local, 'interner> {
         }
     }
 
+    fn elaborate_item<T>(
```

Comment:
Would it be more accurate to call this `elaborate_function`?

Reply:
No, it can elaborate any item that the closure may call. `current_function` is the function's scope to resolve it in. If it is None then the module-level scope is used.